contrib/windows/msys_build.sh

```diff
@@ -206,3 +206,4 @@ cat Make.user
 make -j3 VERBOSE=1
 make build-stats
 #make debug
+make prefix=/c/projects/julia/usr VERBOSE=1 -C examples
```

Comment:
Or we could make `examples/Makefile` refer to the Julia executable from `build_bindir` rather than `bindir`, but I don't know if we want that.